### PR TITLE
Inaprovaline Buff

### PR DESCRIPTION
**Pull request by Diegoflores31**
# About the pull request

Inaprovaline now heals about 0.75 brute and burn to patients that are on hardcrit (sleeping and under 0 health)

# Explain why it's good for the game

For a long time, inaprovaline has been viewed as a risky chemical, often seen as something to avoid in medical kits due to its potential hazards. Many have considered it more of a trap than a useful tool for patient treatment.

When used correctly, this chemical will have the potential to prevent deteriorating health, acting as a stabilizing agent in high-risk scenarios. What was once considered a liability is now proving to be a valuable resource for those looking to treat patients on the brink of dead.

With this change inaprovaline can become a more attractive option for HMs and Mini medics . in the future i plan to add a "tier 2 " version for chemistry to make.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
balance: inaprovaline now heals a small amount of damage over time on critical patients.
/:cl:

### CODE REVIEW

**Comment by hislittlecuzingames** [comment]
"In" is used for oxygen damage, no?

for patients in critical condition, you can use burn and trauma pads to swiftly reduce damage to prevent patients from dying. 

I see IN as a drug specific to a specific use case. 
Dx stops oxygen damage, In reduces oxygen damage. 

Unless I don't know the whole capability of IN. 
Or you mean to expand the use of IN as a general "stabilization" drug

**Comment by Tyranicranger4** [comment]
This change arguably makes it more even more of a liability since the extra healing will give slightly more of a buffer for you to get captured or dragged off somewhere (assuming you have painkillers in you). I don't see this doing much other than making revival mix slightly better - which is already the only decent use case for Inap - *but hey, that's something at least?*

**Comment by cuberound** [comment]
hmm the numbers do not seam to match pr and the description? I think that you heal 0.5 (that is the POTENCY_MULTIPLIER_LOW value) of burn and brute per process

**Comment by Diegoflores31** [comment]
> hmm the numbers do not seam to match pr and the description? I think that you heal 0.5 (that is the POTENCY_MULTIPLIER_LOW value) of burn and brute per process, or possibly 1.5 as list(PROPERTY_CARDIOSTABILIZING = 3) makes me belive it is the potency

Potency as a general rule is HALF the level . This means 3/2 = 1.5/2 = 0.75 
I will probably move inap to be cardiostabilizing lvl 1 or 2 to leave more room for improvement with research healing and possible future chemistry

**Comment by TheManWithNoHands** [comment]
Hard no. Ina is not meant for mini medics or rifelmen, making it more usuable for them is not something that should be aimed for.
Ina is a tool meant to be used by HM and doctors to prevent crit patients to die due to increase of oxy dmg.
It never needed any healing, and never will.

Besides, the "problem"it has is a problem EVERY healing drug has. Being that any form of healing makes it easier to be caped.
Just that Ina does not heal. It prevents taking further oxy dmg, wich will extent the crit counter yes. But it will not heal.
Now the fun part:
Your version makes caps even easier.
And not just because of the heal, but also because of the unconcious part (wich also fucks over its use for HM HARD btw).
What you did, is creat a drug that activly keeps people alive while unconcious. But not so much to make them concious.
What you made, is a drug that keeps crit marines even better alive for caps then base Ina could ever even aproach.
Not even research could cook up something even close to this with tana. Cause tana doesnt deactivate once concious.

You tried to fix, what aint broke, making it worse.

**Comment by Diegoflores31** [comment]
> Hard no. Ina is not meant for mini medics or rifelmen, making it more usuable for them is not something that should be aimed for. Ina is a tool meant to be used by HM and doctors to prevent crit patients to die due to increase of oxy dmg. It never needed any healing, and never will.
> 
> Besides, the "problem"it has is a problem EVERY healing drug has. Being that any form of healing makes it easier to be caped. Just that Ina does not heal. It prevents taking further oxy dmg, wich will extent the crit counter yes. But it will not heal. Ina is one of the least troubelsome drugs in temrs of easier caps.
> 
> Now the fun part: Your version makes caps even easier. And not just because of the heal, but also because of the unconcious part (wich also fucks over its use for HM HARD btw). What you did, is creat a drug that activly keeps people alive while unconcious. But not so much to make them concious. What you made, is a drug that keeps crit marines even better alive for caps then base Ina could ever even aproach. Because it heals brute/burn dmg, prevents oxy dmg gain, and does not heal further once the cap could flee, immediatly triggering the oxy loss. Wich knocks them back out. Not even research could cook up something even close to this with tana. Cause tana doesnt deactivate once concious.
> 
> Buff it if you really want to, but please dont change what activates cardiostabilizing. It fucks to much up, and is a completly uneeded change.

i think there is a missunderstanding . i havent changed anything about cardiostabilizing . the chem works exactly the same as ever . it just has that additional healing factor that triggers when you are inconcious

1) IT IS meant for minimedics . the minimedic pouch ( AKA first responder , comes with inaprovaline) , medkits also used to come with inap ( unsure if they currently still have it )  the goal of this PR is to make inaprovaline more flexible and mark its role as a "use this on crit people to keep the alive untill they can be treated"

2) Making caps easier is completely irrelevant . since the chem already has a painkillers which means merged or not this chemical already allows you to get captured and 0.75 healing will not change that at all.

File: code/modules/reagents/chemistry_properties/prop_positive.dm [hunk 1]
```diff
@@ -904,19 +904,22 @@
...
-	M.pain.apply_pain_reduction(PAIN_REDUCTION_MULTIPLIER * potency)
+	liverino.pain.apply_pain_reduction(PAIN_REDUCTION_MULTIPLIER * potency)
+
+	if(liverino.losebreath >= 10)
+		liverino.losebreath = max(10, liverino.losebreath - 2.5 * potency * delta_time)
 
-	if(M.losebreath >= 10)
-		M.losebreath = max(10, M.losebreath - 2.5 * potency * delta_time)
+	if(liverino.stat == UNCONSCIOUS && liverino.health < 0)
+		liverino.heal_limb_damage(potency * POTENCY_MULTIPLIER_LOW, potency * POTENCY_MULTIPLIER_LOW)
```
Comment (TheManWithNoHands) [inline code/modules/reagents/chemistry_properties/prop_positive.dm:923]: My guy, you ONLY CHANGED the property. You didnt change anything else.

**Comment by TheManWithNoHands** [comment]
> 2\. Making caps easier is completely irrelevant . since the chem already has a painkillers which means merged or not this chemical already allows you to get captured and 0.75 healing will not change that at all.

I thougth Ina making it easier to cap people is the problem your trying to solve. Wich is why i said it only makes it worse in actuallity.
What mysterious problem are you then talking about your trying to solve. 
Assuming everyone knows relevant information for your PR, is how you easily make PRs that wont be understood at all after a few months.

File: code/modules/reagents/chemistry_properties/prop_positive.dm (hunk 1 above)
Comment (TheManWithNoHands) [reply code/modules/reagents/chemistry_properties/prop_positive.dm:923]: You gave it healing, and changed under what condictions it activates.

Comment (Diegoflores31) [reply code/modules/reagents/chemistry_properties/prop_positive.dm:923]: No , you are reading incorrectly . The healing is a plus that only activates when you are in hardcrit.

The old behaviour has not been changed at all please read the code correctly before commenting

Comment (TheManWithNoHands) [reply code/modules/reagents/chemistry_properties/prop_positive.dm:923]: Oh shit, the fucking highligths fucked me over in that regard. Sorry about that.

**Comment by Nanu308** [comment]
I believe medical has it easy enough currently, inaprov also already has a set niche it works very well for imo.

**Comment by Diegoflores31** [comment]
> I believe medical has it easy enough currently, inaprov also already has a set niche it works very well for imo.

may i ask why are you DNM the PRs if there is no change suggestions ? if the PR is denied then close it

**Comment by Drulikar** [comment]
> > I believe medical has it easy enough currently, inaprov also already has a set niche it works very well for imo.
> 
> may i ask why are you DNM the PRs if there is no change suggestions ? if the PR is denied then close it

So another maintainer can either agree or disagree.

I don't think medical needs the additional help this proposes.

**Pull request closed without merging**